data/_nav.yml

```diff
@@ -216,6 +216,37 @@ reference:
         - url: /docs/reference/mpp-dsl-reference.html
           title: Multiplatform Gradle DSL Reference
 
+    - title: Multiplatform Programming (new)
+      description: Reusing code across different platforms
+      content:
+        - url: /docs/reference/mpp-intro.html
+          title: Kotlin Multiplatform
+        - url: /docs/reference/mpp-create-lib.html
+          title: Create a multiplatform library
+        - url: /docs/reference/mpp-discover-project.html
+          title: Discover your project
+        - url: /docs/reference/mpp-share-on-all-platforms.html
+          title: Share code on all platforms
+        - url: /docs/reference/mpp-share-on-multiple-platforms.html
+          title: Share code on multiple platforms
```

Comment:
It might look confusing that there are two points nearby, '_Share code on all platforms_' and '_Share code on multiple platforms_'. I'm not sure what would be better names, though. Maybe '_Share code on similar platforms_'?

Reply:
Combined pages into one - _Share code on platforms_. It seems to me confusing as well.